pyroSAR/drivers.py

```diff
@@ -1173,6 +1173,173 @@ def getCorners(self):
         return self.meta['corners']
 
 
+class EORC_PSR(ID):
+    """
```

Comment:
Please add the driver name to this file:
https://github.com/AntjeUhde/pyroSAR/blob/master/docs/pyroSAR.rst
so it shows up in the documentation.